tests/unit/core/test_installAddon.js

```diff
@@ -601,6 +601,49 @@ describe(__filename, () => {
         .returns(false);
     });
 
+    describe('isAddonEnabled', () => {
+      it('returns add-on status', async () => {
+        const fakeAddonManager = getFakeAddonManagerWrapper();
+        let guid;
+        const addon = createInternalAddon({
+          ...fakeAddon,
+          _addonManager: fakeAddonManager,
```

Comment:
Why would the add-on need a `_addonManager`? That does not look good to me.

Reply:
I've left this in otherwise the assert doesn't work. You also have kept it below so I assume this is okay..?

Reply:
the test case I provided above passes without that so I'd like to see it removed. It is not a good idea to add invalid props.

Reply:
hmm i see this:

```bash
● /Users/rmullin/Sites/www/addons-frontend/tests/unit/core/test_installAddon.js › withInstallHelpers › isAddonEnabled › returns true when the add-on is enabled

    AssertError: expected stub to be called with arguments 

      614 |         const isEnabled = await isAddonEnabled();
      615 | 
    > 616 |         sinon.assert.calledWith(fakeAddonManager.getAddon, addon.guid);
          |                      ^
      617 |         expect(isEnabled).toEqual(true);
      618 |       });
```

you don't see this?